package.json

```diff
@@ -105,7 +105,6 @@
   },
   "dependencies": {
     "@elastic/datemath": "5.0.3",
-    "@elastic/elasticsearch": "7.10.0-rc.1",
```

Comment:
I think you should have a package-lock.json

Reply:
yes. package-lock is changes. @elastic/elasticsearch is gone and @opensearch-project/opensearch is added. but it is ignored [here ](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/3c7e8d5cb00ff184105e7bb1ea8d02350ecefde3/.gitignore#L44)